src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLEncoder;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+
+@WebServlet("/election")
+public class ElectionServlet extends HttpServlet {
+
+  private static final String baseURL = "https://www.googleapis.com/civicinfo/v2/elections?key=";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        StringBuilder strBuf = new StringBuilder();  
+        
+        HttpURLConnection conn=null;
+        BufferedReader reader=null;
+
+        try {  
+            //Declare the connection to civic information API url
+            URL url = new URL(baseURL + "AIzaSyBlNN_uUrbqY2uRaBpKTD-5Jbw6nk8n0_k");  
+            conn = (HttpURLConnection) url.openConnection();  
+
+            conn.setRequestMethod("GET");
+            conn.setRequestProperty("Accept", "application/json");
+            
+            if (conn.getResponseCode() != 200) {
+                throw new RuntimeException("HTTP GET Request Failed with Error code : "
+                              + conn.getResponseCode());
+            }
+            
+            //Read the content from the defined connection
+            //Using IO Stream with Buffer raise highly the efficiency of IO
```

Comment:
nit: please leave a space behind the //.   Also I'm not sure if that comment is grammatically correct, but English is not my forte :P 

This is a good example of "why"--  we want to use BufferedReader for better efficiency :) [though the comment above is more of a "what" comment, and is probably also unnecessary]

Reply:
Sorry -- I was referencing some code I found online and this comment found its way in! Correcting this now.